dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-github-dynamic/src/dynamic/alpha.ts

```diff
@@ -1,14 +1,8 @@
 import { BackendDynamicPluginInstaller } from '@backstage/backend-plugin-manager';
 import catalogModuleGithubEntityProvider from '@backstage/plugin-catalog-backend-module-github/alpha';
-import { catalogModuleGithubOrgEntityProvider } from '@backstage/plugin-catalog-backend-module-github/alpha';
```

Comment:
`catalogModuleGithubOrgEntityProvider` was removed because it was [moved](https://github.com/backstage/backstage/pull/20361) to a [new package](https://github.com/backstage/backstage/tree/master/plugins/catalog-backend-module-github-org).